lane.py

```diff
@@ -800,11 +807,34 @@ def custom_lists_for_identifier(self, list_data_source, list_identifier):
             list_data_source_id = list_data_source.id
         else:
             list_data_source_id = None
+
+        list_featured_works_query = None
         if lists:
             list_ids = [x.id for x in lists]
+            list_featured_works_query = self.extract_list_featured_works_query(lists)
         else:
             list_ids = None
-        return list_data_source_id, list_ids
+        return list_data_source_id, list_ids, list_featured_works_query
+
+    def extract_list_featured_works_query(self, lists):
+        if not lists:
+            return None
+        if isinstance(lists[0], int):
+            # We have CustomList ids instead of CustomList objects.
+            lists = self._db.query(CustomList).filter(CustomList.id.in_(lists))
+            lists = lists.all()
+
+        lists = [custom_list for custom_list in lists if custom_list.featured_works]
+        if not lists:
+            return None
+
+        work_ids = list()
+        for custom_list in lists:
+            work_ids += [work.id for work in custom_list.featured_works]
+
+        works_query = self._db.query(Work).with_labels().\
+            filter(Work.id.in_(work_ids))
+        return works_query
```

Comment:
This is a helpful reminder, thanks. I'll take care of it on the next pass with `CustomList` --> static feeds.